bootcamp-milestone-2/src/components/BlogPreview/BlogPreview.tsx

```diff
@@ -0,0 +1,16 @@
+import Link from "next/link";
+
+export default function BlogPreview({title, content, date, img, slug}: BlogPreviewProps) {
```

Comment:
Instead of having a separate types file, it would be better to have the prop type in the same file imo. This is so whoever is trying to use the component can see the component and expected props all in one file